tests/acceptance/login-test.js

```diff
@@ -113,15 +113,15 @@ test('Log in with Facebook button is visible at load', function(assert) {
 });
 
 test('Successful facebook login redirects', function(assert) {
-  let user = server.create('user');
+  let user = server.create('user', 'facebook');
   registerMockOnInstance(this.application, 'torii-provider:facebook-connect', dummySuccessProviderFb);
   withFeature('socialAuth');
   visit('/login');
 
   click('button:contains(Log in with Facebook)');
 
   andThen(() => {
-    assert.equal(currentURL(), '/');
+    assert.ok(/^index(_loading)?$/.test(currentRouteName()));
```

Comment:
I believe this is fixing something that should be handled by wrapping that click in an `andThen`.